### DISCUSSION TITLE
freenom api续期接口？

### DISCUSSION THREAD

**Discussion by html5syt** [luolongfei/freenom#233] [General]
![Screenshot_20230827_121409_mark via](https://github.com/luolongfei/freenom/assets/53525043/ce1e6b9a-9ff9-4959-9b52-3f4545a99a08)
这好像是freenom的官方api续期接口，目前似乎并没有Captcha，这是否可以作为自动续期的接口？还是我错了？

另外，tg群的封禁机器人是不是太敏感了，我在tg群里发布和上面一样的信息，秒封，请问群主能不能解封一下？名字Tim Green

**Comment by html5syt** [comment]
http://www.freenom.com/en/freenom-api.html

**Reply by luolongfei** [reply]
群里机器人没有 ban 人权限，试试还能不能正常发言

**Reply by html5syt** [reply]
![Screenshot_20230828_095206_org telegram messenger](https://github.com/luolongfei/freenom/assets/53525043/a3bcbe7c-1a47-40b9-9bfe-b0489d7de993)
看看这，还没解封呢，是那个查特皮皮鸡搞得

**Reply by luolongfei** [reply]
现在试试

**Reply by html5syt** [reply]
好了，谢谢

**Comment by luolongfei** [comment]
好像有搞头，空了试试：
curl -X POST https://api.freenom.com/v2/domain/renew.xml\
    -d "domainname=test001.tk\
    &period=3Y\
    &email=john@smith.net\
    &password=68bb651cb1\
    "

**Reply by luolongfei** [reply]
好像不行，使用了正确的账户和密码，依然报错提示账户密码不匹配。然后发现：https://github.com/lpinca/freenom/issues/2#issuecomment-358353709

**Reply by luolongfei** [reply]
只有经销商账户可以使用它们的接口。

**Reply by html5syt** [reply]
那看来还是得手动续期啊😂
不过我看后面有说可以使用v1接口来续期，应该也是受这个限制影响吧。

反正日历多定几个闹钟不会错😂

**Comment by cyao2q** [comment]
目前就tk还能用吧 我就一个tk的唉
